crates/fuel-core/src/graphql_api/worker_service.rs

```diff
@@ -178,9 +181,7 @@ where
         let tx_idx = u16::try_from(tx_idx).map_err(|e| {
             anyhow::anyhow!("The block has more than `u16::MAX` transactions, {}", e)
         })?;
-        let tx_id = tx.cached_id().expect(
-            "The imported block should contains only transactions with cached id",
-        );
+        let tx_id = tx.id(chain_id);
```

Comment:
Why don't we use the cached value anymore?

Reply:
With a WASM executor, we deserialize the result of the execution, and because of that, all metadata is empty.